### PR TITLE
fix: theme toggle in setting

### PR DESCRIPTION
**Pull request by deltork**
### PR Goal? 

Minor patch to the setting menu

### Fixes? 

- Fixed the theme icon button bug
- Enhanced the settings header 

### Feedback sought? 
The theme icon button behaviour and settings dialog display

### Priority? 

Before the next minor release

### Tests added? 

n/a

### How to test? 

Could you test the settings dialog?

### Confidence? 

high

### Version change? 

no

added a line to initialize with the system theme; see issue #38

### CODE REVIEW

**Review by joanise — COMMENTED** [review]
Looks good to me.

The button works correctly, and it looks right, but I don't remember what @roedoejet 's request was exactly, so I'll let him review too to confirm it's fixed.

**Review by joanise — APPROVED** [review]
Alright, I just reviewed Aidan's comments in Slack. It took me quite a while to actually see the difference, but I can now confirm this PR fixes that issue, so approving now, this is ready to rebase-merge.

**Review by roedoejet — APPROVED** [review]
looks good to me! thanks @deltork !

**Pull request merged**
